Firestore/Example/ProtobufCpp.podspec

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A Private podspec for Protobuf which exposes the C++ headers (rather than
+# only the Obj-C headers). Suitable only for use inside this source tree.
+
+Pod::Spec.new do |s|
+  s.name             = 'ProtobufCpp'
+  s.version          = '3.5.2'
+  s.summary          = 'Protocol Buffers v.3 runtime library for C++.'
+  s.homepage         = 'https://github.com/google/protobuf'
+  s.license          = '3-Clause BSD License'
+  s.authors          = { 'The Protocol Buffers contributors' => 'protobuf@googlegroups.com' }
+  s.cocoapods_version = '>= 1.0'
+
+  s.source           = {
+    :git => 'https://github.com/google/protobuf.git',
+    :tag => "v#{s.version}"
+  }
+
+  s.source_files = 'src/**/*.{h,cc}'
+  s.exclude_files = # skip test files. (Yes, the test files are intermixed with
+                    # the source. No there doesn't seem to be a common/simple
+                    # pattern we could use to exclude them; 'test' appears in
+                    # various places throughout the file names and also in a
+                    # non-test file. So, we'll exclude all files that either
+                    # start with 'test' or include test and have a previous
+                    # character that isn't "y" (so that bytestream isn't
+                    # matched.))
+                    'src/**/test*.*',
+                    'src/**/*[^y]test*.*',
```

Comment:
Are there lots of files that match this pattern? If not it may easier for our future selves to understand what we're doing here if we spell them out explicitly.

Reply:
~91.

Unless you're asking about files that would match `*test*` that we want to exclude? (i.e. the files to exclude from the exclude list?) In which case, there's only bytestream.{h,cc}. (But I'm not sure if it's possible to add files back in after excluding them...?)

Reply:
Bleccchhh. Forget I said anything and carry on.